sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -555,6 +572,32 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
       }
     }
   }
+
+  test("Filters should be pushed down for Parquet readers at row group level") {
```

Comment:
@jiangxb1987, here I added a test to make sure  disabling 'spark.sql.parquet.recordFilter' still enables row group level filtering.